test/unit/region.spec.js

```diff
@@ -122,18 +122,17 @@ describe('region', function() {
         onSwapOut: function() {}
       });
 
-      this.MyView = Backbone.View.extend({
+      this.MyView = Marionette.View.extend({
```

Comment:
Why not add a new test as compared to changing this one?

This test had the intent to test compatibility with a vanilla Backbone view, and now this test is gone :crying_cat_face: